ReactiveCocoa/Swift/Property.swift

```diff
@@ -15,6 +15,37 @@ public protocol PropertyType {
 	var signal: Signal<Value, NoError> { get }
 }
 
+extension PropertyType {
+	/// Maps each value in the property to a new value.
+	@warn_unused_result
+	public func map<U>(transform: Value -> U) -> AnyProperty<U> {
+		return AnyProperty(
```

Comment:
Cool. I still think we should extract this to a `lift` function to avoid having each operator repeat the `return AnyProperty(...)` dance.